src/Parsers/ParserCreateQuery.cpp

```diff
@@ -49,6 +49,7 @@ bool ParserNestedTable::parseImpl(Pos & pos, ASTPtr & node, Expected & expected)
 
     auto func = std::make_shared<ASTFunction>();
     tryGetIdentifierNameInto(name, func->name);
+    // FIXME(ilezhankin): func->no_empty_args = true; ?
```

Comment:
Do we allow `Nested` with empty columns list?

Reply:
Looks like no, so this comment is irrelevant.